contrib/create_yaml_files.py

```diff
@@ -0,0 +1,274 @@
+                if expected:
+                    yml_content += "    expected_verdict: " + expected + "\n"
+                if prop[1]:
+                    yml_content += "    subproperty: " + prop[1] + "\n"
+                if "unreach-call" in prop_file and expected == "false":
+                    prop_file = _get_prop(
+                        NAME_TO_PROP_AND_SUBPROP["cover-error"][0], prop_dir, task_dir
+                    )
+                    yml_content += "  - property_file: " + prop_file + "\n"
+        yml_file = curr_task[:-2] + ".yml"
+        with open(yml_file, "w+") as outp:
+            outp.write(yml_content)
+
+        if old_name != curr_task:
+            os.rename(old_name, curr_task)
+            if old_name[-1] == "i":
+                # *.i -> *.c
+                if os.path.exists(old_name[:-1] + "c"):
+                    old_c = old_name[:-1] + "c"
+                # *.c.i -> *.c
+                elif os.path.exists(old_name[:-2]):
+                    old_c = old_name[:-2]
+                # ldv-memsafety/memleaks*.i -> ldv-memsafety/memleaks-notpreprocessed/memleaks*.c
+                elif old_name.startswith("ldv-memsafety/memleaks"):
+                    old_c = (
+                        "ldv-memsafety/memleaks-notpreprocessed/"
+                        + old_name.split("/")[-1][:-1]
+                        + "c"
+                    )
+                else:
+                    old_c = None
+                if old_c:
+                    assert old_c not in all_tasks
+                    curr_task_name = curr_task.split("/")[-1]
+                    new_c_name = (
+                        os.path.dirname(old_c) + "/" + curr_task_name[:-1] + "c"
+                    )
+
+                    os.rename(old_c, new_c_name)
+        for content in sets_to_tasks.values():
+            try:
+                idx = content.index(old_name)
+                content[idx] = yml_file
+            except ValueError:
+                pass
+
+    for task_set, content in sets_to_tasks.items():
+        new_content = list()
+        remaining = set(content)
+        glob_suffix = "*"
+        for task in content:
+            if task not in remaining or task.startswith("#"):
+                continue
+            # get last occurrence of '/'
+            try:
+                last_pathsep = task.rindex("/")
+                prefix_len = last_pathsep + 1
+            except ValueError:
+                prefix_len = 0
+            prefix = task[:prefix_len]
+            globbed_tasks = glob.glob(prefix + glob_suffix)
+            globbed_tasks = [t for t in globbed_tasks if t.endswith(".yml")]
+            assert len(globbed_tasks) > 0
+
+            globbed_tasks = glob.glob(prefix + glob_suffix)
+            globbed_tasks = [t for t in globbed_tasks if t.endswith(".yml")]
+            if not task_set.endswith("testable.set") and (
+                not USE_SUFFIX_WILDCARDS or len(globbed_tasks) > 3 or prefix[-1] == "/"
```

Comment:
`USE_SUFFIX_WILDCARDS` is an undefined identifier.

Reply:
Sorry about that. I removed it.